app/controllers/payment_controller.rb

```diff
@@ -19,9 +19,16 @@ def available_refunds
         intent.payment_record.child_records.charge.map { |record|
           available_amount = record.ruby_amount_available_for_refund
 
+          human_amount_refundable = helpers.ruby_money_to_human_readable(available_amount, record.currency_code)
+          human_amount_payment = helpers.ruby_money_to_human_readable(record.amount_stripe_denomination, record.currency_code)
+
           {
             payment_id: record.id,
-            amount: available_amount,
+            full_amount: record.amount_stripe_denomination,
```

Comment:
You might want to convert this to a Ruby amount...?

Reply:
Depends on what it's used for of course. But if it's really _intentionally_ Stripe denomination, I'd argue the key should be `full_amount_stripe` because all other keys are implicitly Ruby denomination.

Reply:
you are right, this should be ruby (or ISO) amount